src/converters/transactionsConverter.ts

```diff
@@ -141,4 +149,79 @@ export class TransactionsConverter {
             dataItems: dataItems,
         });
     }
+
+    /**
+     * Generally speaking, useful for Relayed V3 transactions.
+     */
+    public transactionOnNetworkToOutcomesOfInnerTransactions(
+        transactionOnNetwork: ITransactionOnNetwork,
+    ): TransactionOutcome[] {
+        const innerTransactions = transactionOnNetwork.innerTransactions || [];
+        const outcomes: TransactionOutcome[] = [];
+
+        for (let index = 0; index < innerTransactions.length; index++) {
+            const innerTransactionAsTransactionOnNetwork = this.convertInnerTransactionToTransactionOnNetwork(
+                transactionOnNetwork,
+                index,
+            );
+
+            const outcome = this.transactionOnNetworkToOutcome(innerTransactionAsTransactionOnNetwork);
+            outcomes.push(outcome);
+        }
+
+        return outcomes;
+    }
+
+    /**
+     * Artificially converts an inner transaction (of a relayed V3) to a transaction on the network,
+     * by matching the inner transaction with its corresponding smart contract result.
+     */
+    private convertInnerTransactionToTransactionOnNetwork(
```

Comment:
We'll keep this private, since it's a temporary workaround, until proper VM output is returned from the Proxy & API.